samples/subsys/sensing/simple/src/main.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys_clock.h>
+#include <zephyr/sensing/sensing.h>
+
+LOG_MODULE_REGISTER(main, LOG_LEVEL_INF);
+
+#define DT_DRV_COMPAT zephyr_sensing
+
+#define DT_SENSOR_TYPE(node)				\
+	{							\
+		.type = DT_PROP(node, sensor_type),		\
+	},							\
+
+struct sensor_type {
+	int32_t type;
+};
+
+static struct sensor_type sensors[] = {
+	DT_FOREACH_CHILD_STATUS_OKAY(DT_DRV_INST(0), DT_SENSOR_TYPE)
+};
+
+static void acc_data_event_callback(sensing_sensor_handle_t handle, const void *buf, int size)
+{
+	const struct sensing_sensor_info *info = sensing_get_sensor_info(handle);
+	struct sensing_sensor_value_3d_int32 *sample = (struct sensing_sensor_value_3d_int32 *)buf;
+
+	ARG_UNUSED(size);
+
+	LOG_INF("%s(%d), handle:%d, Sensor %s data:\t x: %d, y: %d, z: %d",
+			__func__, __LINE__, handle, info->name,
+			sample->readings[0].x,
+			sample->readings[0].y,
+			sample->readings[0].z);
+}
+
+void main(void)
+{
+	const struct sensing_callback_list base_acc_cb_list = {
+		.on_data_event = &acc_data_event_callback,
+	};
+	const struct sensing_callback_list lid_acc_cb_list = {
+		.on_data_event = &acc_data_event_callback,
+	};
+	const struct sensing_sensor_info *info[ARRAY_SIZE(sensors)];
+	struct sensing_sensor_config base_acc_config;
+	struct sensing_sensor_config lid_acc_config;
+	sensing_sensor_handle_t base_acc;
+	sensing_sensor_handle_t lid_acc;
+	int ret, i, num = 0;
+
+	ret = sensing_get_sensors(&num, info);
+	if (ret) {
+		LOG_ERR("sensing_get_sensors error");
+		return;
+	}
+
+	for (i = 0; i < num; ++i) {
+		LOG_INF("Sensor %d: name: %s friendly_name: %s type: %d",
+			 i,
+			 info[i]->name,
+			 info[i]->friendly_name,
+			 info[i]->type);
+	}
+
+	LOG_INF("sensing subsystem run successfully");
+
+	ret = sensing_open_sensor(info[0], &base_acc_cb_list, &base_acc);
+	if (ret) {
+		LOG_ERR("sensing_open_sensor, type:0x%x index:0 error:%d",
+			SENSING_SENSOR_TYPE_MOTION_ACCELEROMETER_3D, ret);
+	} else {
+		base_acc_config.attri = SENSING_SENSOR_ATTRIBUTE_INTERVAL;
+		base_acc_config.interval = 100 * USEC_PER_MSEC;
+		ret = sensing_set_config(base_acc, &base_acc_config, 1);
+		if (ret) {
+			LOG_ERR("base_acc sensing_set_interval error:%d\n", ret);
```

Comment:
When I run this sample, get the following errors:
```
[00:00:00.119,400] <err> main: base_acc sensing_set_interval error:-95
[00:00:00.119,400] <err> main: lid_acc sensing_set_interval error:-95
```
I recommend just deleting the calls to `sensing_set_config()` in this PR.  The samples shouldn't generate errors.

Reply:
not call sensing_set_config() in this PR